phpcs.xml.dist

```diff
@@ -80,13 +80,13 @@
 	<!-- Third-party or auto-generated code -->
 	<exclude-pattern>*/build/*</exclude-pattern>
 	<exclude-pattern>*/data/*</exclude-pattern>
-	<exclude-pattern>tests/e2e/plugins/*</exclude-pattern>
+	<exclude-pattern>packages/e2e-tests/src/plugins/*</exclude-pattern>
 	<exclude-pattern>*/node_modules/*</exclude-pattern>
 	<exclude-pattern>*/third-party/*</exclude-pattern>
 	<exclude-pattern>*/vendor/*</exclude-pattern>
 	<exclude-pattern>assets/js/*.asset.php</exclude-pattern>
 	<exclude-pattern>includes/polyfills/mbstring.php</exclude-pattern>
 	<exclude-pattern>tests/phpstan/*</exclude-pattern>
 	<exclude-pattern>tests/phpunit/includes/MarkupComparison.php</exclude-pattern>
-	<exclude-pattern>tests/e2e/plugins/*</exclude-pattern>
+	<exclude-pattern>packages/e2e-tests/src/plugins/*</exclude-pattern>
```

Comment:
Seems like these are duplicates by accident. Let's emoveone

Could also just do `<exclude-pattern>packages/*</exclude-pattern>` to make things simpler.